client/src/components/DataFiles/CombinedBreadcrumbs/CombinedBreadcrumbs.module.scss

```diff
@@ -3,3 +3,7 @@
   align-items: center;
   gap: 1rem;
 }
+
+.combined-breadcrumbs .breadcrumb-container button {
+  font-size: 1rem;
+}
```

Comment:
**Code not applied.**

This got transpiled as —

```css
._combined-breadcrumbs_croy8_1 ._breadcrumb-container_croy8_7 button {
    font-size: 1rem
}
```

when we expected —

```css
._combined-breadcrumbs_croy8_1 .breadcrumb-container button {
    font-size: 1rem
}
```

**I holiday-hack a fix…**

```suggestion
.combined-breadcrumbs :global(.breadcrumb-container) button {
  font-size: 1rem;
}
```